pkg/cmd/flags/dnscache.go

```diff
@@ -0,0 +1,75 @@
+package flags
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/aquasecurity/tracee/pkg/dnscache"
+)
+
+func dnsCacheHelp() string {
+	return `Select different options for the DNS cache.
+
+Example:
+  --dnscache enable  | enable with default values (see below). Using other options will override them, but is redundant.
+  --dnscache size=X  | will cache up to X dns and ip cross references in the LRU cache (default: 500).
```

Comment:
500 is a very small number, no ? A simple webpage load could solve more than 20 or 30 names depending on images coming from multiple sources (including ads etc). Check the example below, 54 and 82 dns request events in 1 second from a simpple browser session:

```console
$ sudo ./dist/tracee --install-path /tmp/tracee --cache cache-type=mem --cache mem-cache-size=512 --output option:sort-events --output json --output option:parse-arguments --events net_packet_dns_request | linecount 
0 lines/sec
0 lines/sec
6 lines/sec
10 lines/sec
0 lines/sec
2 lines/sec
0 lines/sec
6 lines/sec
7 lines/sec
29 lines/sec
54 lines/sec
82 lines/sec
10 lines/sec
16 lines/sec
1 lines/sec
7 lines/sec
14 lines/sec
3 lines/sec
11 lines/sec
0 lines/sec
18 lines/sec
0 lines/sec
13 lines/sec
17 lines/sec
4 lines/sec
51 lines/sec
36 lines/sec
15 lines/sec
23 lines/sec
2 lines/sec
10 lines/sec
16 lines/sec
5 lines/sec
17 lines/sec
0 lines/sec
10 lines/sec
0 lines/sec
0 lines/sec
0 lines/sec
```

Reply:
These were the default values taken from the original internal implementation, and they went through no tuning, TBH completely guessed. Yes according to this test it should be much more... Perhaps even no capacity and only relying on expiry time? Though i'm worried that would easily cause high memory usage.

Reply:
Yep, I like the expiration. I would do this... calculate the size per "average entry" (a mid sized name, with up to 5 entries, etc). What is the acceptable memory footprint per this feature, and how much names can we fit in ? THat will give us a better choice of default values.

Reply:
I think that if I rework the node implementation (which I will explain why I prefer to do so below), I will rework expiry to be an internal mechanism instead of relying on a global one provided by the LRU. So we can have per item expiry based on actual TTL.

Reply:
I believe this is addressed in the current implementation.